src/mame/drivers/igoabasket.cpp

```diff
@@ -0,0 +1,124 @@
+// license:BSD-3-Clause
+// copyright-holders:
+
+/*
+Skeleton driver for "Basket". Electromechanical machine from the 
+Spanish company IGOA S.A.
+   ____________________________________________________
+  |                          _________                 |
+  |                         | 8xDips  |                |__
+  |  _____________   _________________                  __|
+  | | EMPTY       | | GI AY-3-8910A   |                 __|
+  | |_____________| |_________________|                 __|
+  |  _____________   _________________                  __|
+  | | EMPTY       | | NEC D8155HC     |                 __|
+  | |_____________| |_________________|                 __|                
+  |  _____________   _______  _________      ________   __|
+  | | EPROM       | |DM7417N  SN74LS373N    |ULN2003R|  __|
+  | |_____________|  _________________       ________   __|
+  |   ____________  | M5L8085AP       |     |ULN2003A|  __|
+  |  | HM6116LP-3 | |_________________|                |
+  |  |____________|           ________                 |__
+  |   ________         Xtal   SN74LS14N                 __|
+  |   82S123AN     6.144 MHz  _________________         __|
+  |   ________               | NEC D8155HC     |        __|
+  |   74LS393N               |_________________|        __|
+  |   ________    ________                              __|
+  |  PAL16R4ACN  |TC4011BP|                             __|
+  |                  _____                              __|
+  |  __________     NE555N             ________         __|
+  |  | BATT 5V |                      |ULN2803A         __|
+  |  |_________|                                        __|
+  |     __                                             |
+  |    (__) <- Button                                  |
+  |____________________________________________________|
+
+*/
+
+#include "emu.h"
+#include "emupal.h"
```

Comment:
emupal.h, unused header

Reply:
Fixed, thanks again.

One question: Should I remove it too from missbamby,cpp? (I took that driver as a template, that's why it was there)
Also, do you think this new machine should be added to missbamby instead of on a new driver? missbamby currently contains several slot machines and a pinball.

Thx!

Reply:
If missbamby doesn't use emupal.h, yes remove it.
The PCB seems very similar to me, the main difference is basket has 1 more 8155 chip? I wouldn't know how different it is overall, until emulating it. But I think it's safe to just place it in missbamby.cpp for now.